### DISCUSSION TITLE
What are partitions and how do I fix no partitions error?

### DISCUSSION THREAD

**Discussion by yuk75** [BadDogSkyrim/PyNifly#227] [Q&A]
![image](https://github.com/BadDogSkyrim/PyNifly/assets/6256167/36a17418-b208-4c78-96ea-846d1e68003c)
![image](https://github.com/BadDogSkyrim/PyNifly/assets/6256167/f7eae453-28a4-4d6a-8ae7-5e860754288f)

I assume this happened because I had to split the pants seam....but I dunno.

**Comment by BadDogSkyrim** [comment]
https://github.com/BadDogSkyrim/PyNifly/wiki/FO4-Segments-and-Subsegments

It's a PITA, honestly. Look at your nif in face-select mode with "show weight" on (it's under "Mesh Overlays". Then you can click the different groups. You want each face to have all its verts in one group, no overlap with another group, and no face partly in one group and partly in another. If you're working with quads you may have to triangluate along the seam. 

The "No Partitions" error should be fixed. The "Multiple Partitions" error you can usually get away with. Just make sure it doesn't crash the game.
